.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -42,7 +42,8 @@ val gradleParameters = listOf(
         "--continue",
         """-I "%teamcity.build.checkoutDir%/gradle/init-scripts/build-scan.init.gradle.kts"""",
         java7Homes[OS.linux]!!,
-        java9Homes[OS.linux]!!
+        java9Homes[OS.linux]!!,
+        "-Dorg.gradle.internal.tasks.createops"
```

Comment:
Do you need to set the property to something? Or is it enough to set the property at all?

Reply:
You can leave it empty or set it to a boolean value. Check here for the implementation: https://github.com/gradle/gradle/pull/5655/files#diff-c1f28c1e4f6ce6dd164dd64d233b8f2fR208